test/visual_regression/services/visual_testing/take_percy_snapshot.js

```diff
@@ -20,18 +20,44 @@
 import { readFileSync } from 'fs';
 import { agentJsFilename } from '@percy/agent/dist/utils/sdk-utils';
 
-export function takePercySnapshot() {
+export function takePercySnapshot(show, hide) {
   if (!window.PercyAgent) {
     return false;
   }
 
+  const queryAll = selector => [
+    ...document.querySelectorAll(selector)
+  ];
+
+  const setSelectorClass = (selectors, className, remove) => {
```

Comment:
uber minor nitpick, but I always try to avoid naked boolean arguments, especially when they essentially fork the behavior of the function to do two different things. I'd prefer to see this implemented as two functions: `add(selectors, className)` and `remove(selectors, className)`